HLTrigger/Egamma/src/HLTElectronMuonInvMassFilter.cc

```diff
@@ -102,8 +115,18 @@ HLTElectronMuonInvMassFilter::hltFilter(edm::Event& iEvent, const edm::EventSetu
     pElectron.push_back( pThisEle );
     eleCharge.push_back( refele->charge() );
   }
+  
+  for(unsigned int i=0;i<clusCands.size();i++){
```

Comment:
please use reserve on pElectron and eleCharge  to pre-allocate the memory here.
(eg, pElectron.reserve(clusCands.size()); )